### DISCUSSION TITLE
PNP Search Filter - Supporting "True"/ "False" or IsDocument as a filter

### DISCUSSION THREAD

**Discussion by bkvwilson** [microsoft-search/pnp-modern-search#1706] [General]
Hi,

I configured a filter using IsDocument as the filter field, the goal being to show a toggle to the user to show documents only. I have tried both Combo Box and Check Box but it doesnt display. I confirm that the results webpart has this property selected. 

Does the PNP Search Filters web part support true/false type or do i have to map this to a refinablestring and use that?

Great work on the web parts and thanks for your help! 
Brian

**Comment by FranckyC** [comment]
Hi @bkvwilson , **IsDocument** property is not refinable by default so you can't use it as filter. You should use a `RefinableStringXX` placeholder property and map the same crawled properties:

![image](https://user-images.githubusercontent.com/5659122/149169460-d7ff8de3-1da6-498e-a8ed-1c608d978065.png)

**Reply by wobba** [reply]
I want to call out that there is a Microsoft API bug today where you can use any managed property as your filter property, but expect this to go away once it's fixed.
